suite-common/wallet-config/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import { networks, NetworkSymbol } from './networksConfig';
+
+export type EthereumTokenSymbol = string & { __type: 'EthereumTokenSymbol' };
```

Comment:
Please create new package `@suite-native/ethereum-tokens` and put it there. We will use this package in vol.2 task.

Reply:
https://github.com/trezor/trezor-suite/pull/7676/commits/4961fc7a9a7d0a7573924322467aabdd9d7e8b05